objgraph.py

```diff
@@ -29,7 +29,7 @@
 __author__ = "Marius Gedminas (marius@gedmin.as)"
 __copyright__ = "Copyright (c) 2008-2014 Marius Gedminas"
 __license__ = "MIT"
-__version__ = "1.8.2.dev0"
+__version__ = "1.9.0dev0"
```

Comment:
I believe I asked for "1.9.0.dev0".  Recent setuptools (or pip?) emits warnings when the version number isn't PEP 440-compliant, and [PEP-440](https://www.python.org/dev/peps/pep-0440/) requires the silly dot in front of "dev".